src/locales/en/main.json

```
@@ -4957,6 +4981,117 @@
     "processingVideo": "Processing video…",
     "running": "Running…"
   },
+  "agent": {
```

Biome (2.5.7)

[error] The key agent was already declared.

(lint/suspicious/noDuplicateObjectKeys)